### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by saade** [ritz078/ngEmbed#85]
The emojis is working fine, links too, but embeds not, any ideas?

![image](https://cloud.githubusercontent.com/assets/14329460/23265004/14ebe3c2-f9c2-11e6-8ac6-03376223e441.png)
![image](https://cloud.githubusercontent.com/assets/14329460/23265015/205ea924-f9c2-11e6-8706-489172a9e8da.png)
![image](https://cloud.githubusercontent.com/assets/14329460/23265023/27980b68-f9c2-11e6-8318-d469a2b8b3a4.png)

**Comment by saade** [comment]
Oh, sorry. I didn't saw the "Its features are limited to converting text into emojis, font smileys and HTML Links." on the filter option.

:)

**Comment by ritz078** [comment]
twitter, youtube and spotify should work.

**Comment by saade** [comment]
Yes, they are working now. But with millions errors in console, like:

![image](https://cloud.githubusercontent.com/assets/14329460/23301429/a99816ca-fa68-11e6-962c-9d4ba1d0fe83.png)

**Comment by saade** [comment]
You are using angular.element the wrong way, at:

```
 /**
                             * Adding line numbers to code
                             */
                            $timeout(function () {
                                if (options.code.lineNumbers) {
                                    angular.element('.ne-code').each(function () {
```

can i send a pull request?

**Comment by ritz078** [comment]
PR is always welcome.

**Comment by robman87** [comment]
If jQuery is loaded there shouldn't be any errors, however it would be better to implement without jQuery as dependency.
